microsoft-edge/devtools-guide-chromium/console/live-expressions.md

```diff
@@ -48,15 +48,15 @@ To add a live expression:
 
    <!-- revise the step & the capture after it: -->
 
-1. In that same window, go to the webpage [Overview of DevTools](../overview.md), and then click the **Docs** button in the upper left.
+1. In that same window, go to the webpage [Overview of DevTools](../overview.md), and then click the **Docs** button in the upper right.
```

Comment:
we might delete this line later, but I think the intended "Docs" button is upper left, I don't see a "Docs" button in right.

![image](https://user-images.githubusercontent.com/45407486/184279869-e6a319a5-e380-4ada-a434-02ce45fbdc30.png)

Reply:
I also don't have the button in the upper right.
But, I agree with you both, using several demo sites to show the same thing, and especially using DMC, is not a good idea. So I'll log a task for us to rewrite this page. For now we can merge this as the new images look great. Thanks Josh.